### ISSUE TITLE
Timing out when localstack is starting

### ISSUE THREAD

**Issue by cwagnello** [localstack/helm-charts#62]
When localstack is starting up in my microk8s cluster it is not able to successfully start up. It's trying to download/install some things but it only has 30 seconds to do it. This usually fails and it gets into a loop of just constantly doing this over and over again. 

Two questions I have: 

1. Could we expose a variable for the helm-chart to increase the timeout so it has the time to download and install all of the extra stuff? 
2. Is the downloading/installing a bug with starting up localstack, meaning it's not supposed to be downloading all this extra stuff?

This was working before and I think this might benefit anyone else who can't get localstack started in a local k8s.

```
Waiting for all LocalStack services to be ready
2022-09-26 22:11:48,852 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-09-26 22:11:48,853 INFO supervisord started with pid 15
2022-09-26 22:11:49,865 INFO spawned: 'infra' with pid 20
SERVICES variable is ignored if EAGER_SERVICE_LOADING=0.

LocalStack version: 1.1.1.dev
LocalStack build date: 2022-09-14
LocalStack build git hash: 1cf9fd4a

2022-09-26T22:11:50.773 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='_patch_botocore_json_parser', value='localstack.aws.client:_patch_botocore_json_parser', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.774 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='_run_init_scripts_on_start', value='localstack.runtime.init:_run_init_scripts_on_start', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.775 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='register_partition_adjusting_proxy_listener', value='localstack.plugins:register_partition_adjusting_proxy_listener', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.775 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='add_aws_request_logger', value='localstack_ext.analytics.plugins:add_aws_request_logger', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.775 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='add_custom_edge_routes', value='localstack_ext.plugins:add_custom_edge_routes', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.777 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='add_iam_enforcement_listener', value='localstack_ext.services.iam.plugins:add_iam_enforcement_listener', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.777 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='add_legacy_iam_enforcement_listener', value='localstack_ext.services.iam.plugins:add_legacy_iam_enforcement_listener', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.777 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='extensions_on_infra_start', value='localstack_ext.extensions.plugins:extensions_on_infra_start', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='init_stackinfo_handler', value='localstack_ext.services.internal.stackinfo.plugins:init_stackinfo_handler', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='patch_account_id_resolver', value='localstack_ext.plugins:patch_account_id_resolver', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='patch_get_account_id_from_access_key_id', value='localstack_ext.plugins:patch_get_account_id_from_access_key_id', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='register_pods_api', value='localstack_ext.persistence.plugins:register_pods_api', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='register_state_serializer_handlers', value='localstack_ext.persistence.plugins:register_state_serializer_handlers', group='localstack.hooks.on_infra_start')
2022-09-26T22:11:50.778 DEBUG --- [  MainThread] stevedore.extension        : found extension EntryPoint(name='setup_pro_infra', value='localstack_ext.plugins:setup_pro_infra', group='localstack.hooks.on_infra_start')

...

2022-09-26T22:11:50.779 DEBUG --- [  MainThread] plugin.manager             : plugin localstack.hooks.on_infra_start:setup_pro_infra is disabled
2022-09-26T22:11:50.779 DEBUG --- [  MainThread] localstack.runtime.init    : Init scripts discovered: {BOOT: [], START: [], READY: [], SHUTDOWN: []}
2022-09-26T22:11:50.806  INFO --- [d-2 (worker)] l.services.install         : Downloading and installing local Installing lambda runtime server. This may take some time.
2022-09-26T22:11:50.811  INFO --- [-23 (worker)] l.services.install         : Downloading and installing local KMS server. This may take some time.
2022-09-26T22:11:50.815 DEBUG --- [-17 (worker)] localstack.packages.core   : Starting installation of dynamodb-local...
2022-09-26T22:11:50.816 DEBUG --- [-17 (worker)] localstack.utils.run       : Executing command: ['unzip', '-o', '-q', '/tmp/localstack.ddb.zip']
2022-09-26T22:11:50.817 DEBUG --- [-15 (worker)] l.services.install         : getting kinesis-mock for linux aarch64
2022-09-26T22:11:51.195 DEBUG --- [-17 (worker)] localstack.utils.run       : Executing command: ['zip', '-u', 'DynamoDBLocal.jar', 'log4j2.xml']
2022-09-26 22:11:51,195 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Waiting for all LocalStack services to be ready
2022-09-26T22:11:56.058 DEBUG --- [d-3 (worker)] localstack.utils.http      : Starting download from https://raw.githubusercontent.com/LukeMizuhashi/cfn-response/master/index.js to /opt/code/localstack/.filesystem/usr/lib/localstack/lambda.cfn-response.js (878 bytes)
2022-09-26T22:11:56.060 DEBUG --- [d-3 (worker)] localstack.utils.http      : Done downloading https://raw.githubusercontent.com/LukeMizuhashi/cfn-response/master/index.js, response code 200, total bytes 1951
2022-09-26T22:11:56.157 DEBUG --- [d-2 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/lambda-runtime-init/releases/download/v0.1.1-pre/aws-lambda-rie-arm64 to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie (9236354 bytes)
2022-09-26T22:11:56.326  INFO --- [-15 (worker)] l.services.install         : downloading kinesis-mock binary from https://github.com/etspaceman/kinesis-mock/releases/download/0.2.5/kinesis-mock.jar
2022-09-26T22:11:56.511 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 1048576) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:11:56.675 DEBUG --- [-17 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/388cd73f45bfd3bcf7ad40aa35499093061c7962/dynamodb-local-patch/target/ddb-local-loader-0.1.jar to /opt/code/localstack/.filesystem/var/lib/localstack/lib/dynamodb-local/latest/ddb-local-loader-0.1.jar (4226 bytes)
2022-09-26T22:11:56.677 DEBUG --- [-17 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/388cd73f45bfd3bcf7ad40aa35499093061c7962/dynamodb-local-patch/target/ddb-local-loader-0.1.jar, response code 200, total bytes 4226
2022-09-26T22:11:56.716 DEBUG --- [-23 (worker)] localstack.utils.http      : Starting download from https://s3-eu-west-2.amazonaws.com/local-kms/3/local-kms_linux-arm64.bin to /opt/code/localstack/.filesystem/usr/lib/localstack/kms/local-kms.linux-arm64.bin (14559534 bytes)
2022-09-26T22:11:56.796 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 2097152) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:11:56.902 DEBUG --- [-17 (worker)] localstack.utils.http      : Starting download from https://repo1.maven.org/maven2/org/javassist/javassist/3.28.0-GA/javassist-3.28.0-GA.jar to /opt/code/localstack/.filesystem/var/lib/localstack/lib/dynamodb-local/latest/javassist.jar (851531 bytes)
2022-09-26T22:11:56.953 DEBUG --- [-15 (worker)] localstack.utils.http      : Starting download from https://github.com/etspaceman/kinesis-mock/releases/download/0.2.5/kinesis-mock.jar to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar (43288236 bytes)
2022-09-26T22:11:59.816 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 3145728) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
Waiting for all LocalStack services to be ready
2022-09-26T22:12:04.730 DEBUG --- [-17 (worker)] localstack.utils.http      : Done downloading https://repo1.maven.org/maven2/org/javassist/javassist/3.28.0-GA/javassist-3.28.0-GA.jar, response code 200, total bytes 851531
2022-09-26T22:12:05.381 DEBUG --- [-17 (worker)] localstack.utils.http      : Starting download from https://repo1.maven.org/maven2/org/slf4j/slf4j-ext/1.8.0-beta4/slf4j-ext-1.8.0-beta4.jar to /opt/code/localstack/.filesystem/var/lib/localstack/lib/dynamodb-local/latest/DynamoDBLocal_lib/slf4j-ext-1.8.0-beta4.jar (38652 bytes)
2022-09-26T22:12:05.550 DEBUG --- [-17 (worker)] localstack.utils.http      : Done downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-ext/1.8.0-beta4/slf4j-ext-1.8.0-beta4.jar, response code 200, total bytes 38652
2022-09-26T22:12:05.551 DEBUG --- [-17 (worker)] localstack.utils.run       : Executing command: ['unzip', '-o', '/opt/code/localstack/.filesystem/var/lib/localstack/lib/dynamodb-local/latest/DynamoDBLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:05.553 DEBUG --- [-17 (worker)] localstack.utils.run       : Executing command: ['zip', 'DynamoDBLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:05.580 DEBUG --- [-17 (worker)] localstack.packages.core   : Installation of dynamodb-local finished.
2022-09-26T22:12:06.855 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: rm -rf "/opt/code/localstack/.filesystem/usr/lib/localstack/home"
2022-09-26T22:12:06.968 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 1048576) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:07.049 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 4194304) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:12:07.062 DEBUG --- [-23 (worker)] localstack.utils.http      : Written 1048576 bytes (total 1048576) to /opt/code/localstack/.filesystem/usr/lib/localstack/kms/local-kms.linux-arm64.bin
2022-09-26T22:12:07.447 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/com/amazonaws/stepfunctions/local/runtime/Config.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/com/amazonaws/stepfunctions/local/runtime/Config.class (16268 bytes)
2022-09-26T22:12:07.455 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/com/amazonaws/stepfunctions/local/runtime/Config.class, response code 200, total bytes 16268
2022-09-26T22:12:07.456 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'com/amazonaws/stepfunctions/local/runtime/Config.class']
2022-09-26T22:12:07.758 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/com/amazonaws/stepfunctions/local/runtime/executors/task/LambdaTaskStateExecutor.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/com/amazonaws/stepfunctions/local/runtime/executors/task/LambdaTaskStateExecutor.class (19963 bytes)
2022-09-26T22:12:07.769 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/com/amazonaws/stepfunctions/local/runtime/executors/task/LambdaTaskStateExecutor.class, response code 200, total bytes 19963
2022-09-26T22:12:07.771 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'com/amazonaws/stepfunctions/local/runtime/executors/task/LambdaTaskStateExecutor.class']
2022-09-26T22:12:07.966 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 5242880) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:12:08.027 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 2097152) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:08.269 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/RegionAspect.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/cloud/localstack/RegionAspect.class (10777 bytes)
2022-09-26T22:12:08.275 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/RegionAspect.class, response code 200, total bytes 10777
2022-09-26T22:12:08.276 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'cloud/localstack/RegionAspect.class']
2022-09-26T22:12:08.592 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 6291456) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:12:08.696 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 3145728) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:08.727 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/StepFunctionsStarter.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/cloud/localstack/StepFunctionsStarter.class (1688 bytes)
2022-09-26T22:12:08.729 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/StepFunctionsStarter.class, response code 200, total bytes 1688
2022-09-26T22:12:08.731 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'cloud/localstack/StepFunctionsStarter.class']
2022-09-26T22:12:09.040 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/Async2ServiceApi.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/cloud/localstack/Async2ServiceApi.class (1118 bytes)
2022-09-26T22:12:09.043 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/Async2ServiceApi.class, response code 200, total bytes 1118
2022-09-26T22:12:09.045 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'cloud/localstack/Async2ServiceApi.class']
2022-09-26T22:12:09.154 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 7340032) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
2022-09-26T22:12:09.298 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 4194304) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:09.379 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/DescribeExecutionParsed.class to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/cloud/localstack/DescribeExecutionParsed.class (6409 bytes)
2022-09-26T22:12:09.383 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/cloud/localstack/DescribeExecutionParsed.class, response code 200, total bytes 6409
2022-09-26T22:12:09.384 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'cloud/localstack/DescribeExecutionParsed.class']
2022-09-26T22:12:09.715 DEBUG --- [d-2 (worker)] localstack.utils.http      : Written 1048576 bytes (total 8388608) to /opt/code/localstack/.filesystem/usr/lib/localstack/aws-lambda-rie
Waiting for all LocalStack services to be ready
2022-09-26T22:12:09.825 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/META-INF/aop.xml to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/META-INF/aop.xml (141 bytes)
2022-09-26T22:12:09.828 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/ac84739adc87ff4b5553478f6849134bcd259672/stepfunctions-local-patch/META-INF/aop.xml, response code 200, total bytes 234
2022-09-26T22:12:09.830 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'META-INF/aop.xml']
2022-09-26T22:12:09.836 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['unzip', '-o', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:09.838 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', 'StepFunctionsLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:09.845 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['unzip', '-o', '/opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/StepFunctionsLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:09.847 DEBUG --- [-22 (worker)] localstack.utils.run       : Executing command: ['zip', 'StepFunctionsLocal.jar', 'META-INF/MANIFEST.MF']
2022-09-26T22:12:09.877 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 5242880) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:10.027 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://repo1.maven.org/maven2/org/aspectj/aspectjrt/1.9.7/aspectjrt-1.9.7.jar to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/aspectjrt-1.9.7.jar (122874 bytes)
2022-09-26T22:12:10.175 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://repo1.maven.org/maven2/org/aspectj/aspectjrt/1.9.7/aspectjrt-1.9.7.jar, response code 200, total bytes 122874
2022-09-26T22:12:10.305 DEBUG --- [d-2 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/lambda-runtime-init/releases/download/v0.1.1-pre/aws-lambda-rie-arm64, response code 200, total bytes 9236354
2022-09-26T22:12:10.352 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://repo1.maven.org/maven2/org/aspectj/aspectjweaver/1.9.7/aspectjweaver-1.9.7.jar to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/aspectjweaver-1.9.7.jar (2087074 bytes)
2022-09-26T22:12:10.517 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 6291456) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:10.688 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 1048576) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/aspectjweaver-1.9.7.jar
2022-09-26T22:12:10.985 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1038498 bytes (total 2087074) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/aspectjweaver-1.9.7.jar
2022-09-26T22:12:10.989 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://repo1.maven.org/maven2/org/aspectj/aspectjweaver/1.9.7/aspectjweaver-1.9.7.jar, response code 200, total bytes 2087074
2022-09-26T22:12:11.556 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 7340032) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:11.691 DEBUG --- [-22 (worker)] localstack.utils.http      : Starting download from https://github.com/localstack/localstack-artifacts/raw/a4adc8f4da9c7ec0d93b50ca5b73dd14df791c0e/stepfunctions-internal-awssdk/awssdk.zip to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip (8373983 bytes)
2022-09-26T22:12:12.086 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 1048576) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:12.437 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 2097152) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:12.666 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 8388608) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:12.792 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 3145728) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:13.160 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 4194304) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:13.539 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 5242880) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:13.714 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 9437184) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:13.966 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 6291456) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:14.407 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1048576 bytes (total 7340032) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:14.511 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 10485760) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:14.852 DEBUG --- [-22 (worker)] localstack.utils.http      : Written 1033951 bytes (total 8373983) to /opt/code/localstack/.filesystem/usr/lib/localstack/stepfunctions/localstack-internal-awssdk/awssdk.zip
2022-09-26T22:12:14.861 DEBUG --- [-22 (worker)] localstack.utils.http      : Done downloading https://github.com/localstack/localstack-artifacts/raw/a4adc8f4da9c7ec0d93b50ca5b73dd14df791c0e/stepfunctions-internal-awssdk/awssdk.zip, response code 200, total bytes 8373983
2022-09-26T22:12:14.928 DEBUG --- [-23 (worker)] localstack.utils.http      : Written 1048576 bytes (total 2097152) to /opt/code/localstack/.filesystem/usr/lib/localstack/kms/local-kms.linux-arm64.bin
2022-09-26T22:12:15.162 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 11534336) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:15.680 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 12582912) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:16.121 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 13631488) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
2022-09-26T22:12:16.509 DEBUG --- [-15 (worker)] localstack.utils.http      : Written 1048576 bytes (total 14680064) to /opt/code/localstack/.filesystem/usr/lib/localstack/kinesis-mock/kinesis-mock.jar
Waiting for all LocalStack services to be ready
Sending -15 to supervisord
2022-09-26 22:12:16,863 WARN received SIGTERM indicating exit request
2022-09-26 22:12:16,863 INFO waiting for infra to die
2022-09-26 22:12:17,747 INFO stopped: infra (exit status 0)
Stream closed EOF for default/avail-localstack-69cc4f7575-bpvbj (localstack)
```

**Comment by navneel99** [comment]
Facing the same issue.

**Comment by cwagnello** [comment]
@navneel99 I fixed the timeout issue by overriding the `livenessProbe.periodSeconds` value listed in the link below.

https://github.com/localstack/helm-charts/blob/main/charts/localstack/values.yaml#L74-L79

livenessProbe.periodSeconds default is set to 10 I bumped it up to 30 seconds. When localstack starts up and does the download/installing it now has enough time. In my last restart of the deployment it took about 65 seconds for localstack to fully come up. I hope that helps.
